spec/controllers/lists_controller_spec.rb

```diff
@@ -12,7 +12,7 @@
 
   describe '#create' do
     it "creates a list and checks it's title" do
-      post :create, list: { title: 'test' }
+      post :create, list: { title: 'test', user_id: @user }
```

Comment:
params should not need to contain `user_id` here, the `create` action will know the signed in user via `current_user`. 

The previous version of this line looked more correct:

```ruby
post :create, list: { title: 'test' }
```